buyer-api/src/utils/funding.js

```diff
@@ -0,0 +1,30 @@
+import { BigNumber } from 'bignumber.js';
```

Comment:
I would also move this into `transfersFacade` for its dependencies.

Reply:
Saw this comment after merging 😑 
I'll add the fix for this in the next PR